src/modules/endpoint/reducers/cityContentReducer.js

```diff
@@ -6,51 +6,86 @@ import morphContentLanguage from './morphContentLanguage'
 import pushCategory from './pushCategory'
 import pushEvent from './pushEvent'
 import type { CityContentActionType } from '../../app/StoreActionType'
-import initializeCityContent from './initializeCityContent'
+import createCityContent from './createCityContent'
 
 export default (
   state: CityContentStateType | null = defaultCityContentState, action: CityContentActionType
 ): CityContentStateType | null => {
-  if (action.type === 'INITIALIZE_CITY_CONTENT') {
-    return initializeCityContent(state, action)
-  }
-
-  if (!state) {
-    return defaultCityContentState
-  }
-
   switch (action.type) {
     case 'SWITCH_CONTENT_LANGUAGE':
-      return { ...state, switchingLanguage: true }
+      return state === null ? null : { ...state, switchingLanguage: true }
+    case 'PUSH_LANGUAGES':
+      if (state === null) {
```

Comment:
There is quite some duplicate code about checking for `state === null`.

We could split this split case (first part needs `state !== null`) such that each part is in a separate function. Then we can check at the start of the function for the state and throw an error.

Reply:
I will leave it like this, since I think we should check this for each action, to make it more explicit...